docs/execute/requirements.md

```diff
@@ -7,7 +7,7 @@ You''ll need a bash based terminal, docker and python3.12
 ```
 sudo add-apt-repository ppa:deadsnakes/ppa
 sudo apt update
-sudo apt install python3.12 python3.12-distutils python3.12-venv
+sudo apt install python3.12 python3.12-distutils python3.12-venv python3.12-dev
```

Comment:
why this change?

Reply:
IIRC, I had issue while running `./build.sh`.
`pip` was not able to install some packages